AdvanceSpecFlowProject/SpecFlowProject/StepDefinitions/RegistrationStepDefinitions.cs

```diff
@@ -0,0 +1,77 @@
+using SpecFlowProject.Hooks;
+using SpecFlowProject.Model;
+using SpecFlowProject.Utilities;
+using TechTalk.SpecFlow;
+using MarsAdvanceTaskNUnit.Steps;
+using NUnit.Framework;
+
+namespace SpecFlowProject.StepDefinitions
+{
+    [Binding]
+    public class RegistrationStepDefinitions : CommonDriver
+    {
+        RegistrationSteps RegistrationStepsObj;
```

Comment:
same comments apply to all step definition which i have mentioned

Reply:
Issue resolved. Please review.